src/test/regress/expected/multi_explain.out

```diff
@@ -1357,31 +1357,33 @@ WITH cte1 AS (SELECT * FROM cte1 WHERE EXISTS (SELECT * FROM cte1) LIMIT 5)
 SELECT s FROM cte1 WHERE EXISTS (SELECT * FROM cte1);
 Custom Scan (Citus INSERT ... SELECT)
   INSERT/SELECT method: pull to coordinator
-  ->  Result
-        One-Time Filter: $3
+  ->  Subquery Scan on citus_insert_select_subquery
```

Comment:
This is because subquery wrapping happens for this query. I think this is better than better, since it matches the actual queries which are being sent to workers.